portfolio/src/main/webapp/testing.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This looks like it was for local testing, don't check it into git

Reply:
Is there a way to remove the file from git or should I just stop adding it when I commit?

Reply:
Yes, "git rm" will delete the file.  You'll need to do a new commit removing it.  Ideally you would catch this before adding it, because it pollutes the commit history

Reply:
Resolving because I stopped checking testing.html into git

Reply:
You need to delete it from git too